packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -80,11 +81,21 @@ router.get(
  * POST /internal/carequality/directory/organization
  *
  * Creates or updates the organization in the Carequality Directory.
+ * Providing no body will use the env vars for the organization details.
  */
 router.post(
   "/directory/organization",
   asyncHandler(async (req: Request, res: Response) => {
-    await createOrUpdateCQOrganization();
+    const body = req.body;
+    let orgDetails;
+    if (body.oid) {
+      orgDetails = cqOrgDetailsSchema.parse(body);
+    } else {
+      const orgDetailsString = Config.getCQOrgDetails();
+      orgDetails = cqOrgDetailsSchema.parse(JSON.parse(orgDetailsString));
+    }
```

Comment:
I think this relates to your comment about whether to have Metriport's org details on config, right? Yeah, this is odd and I'd just get the full payload from the request.

Reply:
Yea exactly! I thought this change would make sense - shoulda just gone with it straight away haha